table/block_based_table_reader.cc

```diff
@@ -2406,6 +2407,28 @@ Status BlockBasedTable::Get(const ReadOptions& read_options, const Slice& key,
           break;
         }
 
+        bool hash_effective, found;
+        biter.SeekForGet(key, &hash_effective, &found);
```

Comment:
Can we do a check here for data block index type before calling SeekForGet, so that this call can be avoided in the normal case?